notices.json

```diff
@@ -37,21 +37,21 @@
     }
   },
   {
-    "id": "server_upgrade_v5.35",
+    "id": "server_upgrade_v5.36",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<5.35"],
+      "serverVersion": ["<5.36"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-05-21T00:00:00Z"
+      "displayDate": ">= 2021-06-21T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "The new feature release offers shared channels, stakeholder overview and ad hoc tasks for incident collaboration, file search, and more. [Please also see the important upgrade notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).",
+        "description": "The new feature release offers updates to Incident Collaboration with enhanced permission controls and automated welcome messages, and the addition of Hungarian language support. Focalboard project management is now integrated with Mattermost for the first time as a beta feature. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```

Comment:
@amyblais I would propose adding @itao and @chenilim to review this, given this release is mainly about their features.